src/plugins/elasticsearch.js

```diff
@@ -15,14 +15,16 @@ function createWrapRequest (tracer, config) {
           'service.name': config.service || `${tracer._service}-elasticsearch`,
           'resource.name': `${params.method} ${quantizePath(params.path)}`,
           'span.type': 'elasticsearch',
-          'elasticsearch.url': params.path,
+          'elasticsearch.url': quantizePath(params.path),
```

Comment:
It's ok to have actual values as tags, assuming they have a reasonable length.

Reply:
Wouldn't this cause high-dimensionality? This function only replaces numbers with '?'

Reply:
High cardinality isn't a concern on tags; only on operation names.